src/data/finset/noncomm_prod.lean

```diff
@@ -183,6 +189,23 @@ given a proof that `+` commutes on all elements `f x` for `x ∈ s`."]
 def noncomm_prod (s : finset α) (f : α → β) (comm : ∀ (x ∈ s) (y ∈ s), commute (f x) (f y)) : β :=
 (s.1.map f).noncomm_prod (by simpa [multiset.mem_map, ←finset.mem_def] using comm)
 
+/-- A helper for rewriting the dependent arguments. -/
+@[to_additive "A helper for rewriting the dependent arguments "]
```

Comment:
```suggestion
@[congr, to_additive "A helper for rewriting the dependent arguments "]
```
I don't know if it makes sense to tag the others, since it looks like the generic congruence matches them just fine.